hazelcast/src/main/java/com/hazelcast/spi/impl/OperationScheduler.java

```diff
@@ -0,0 +1,49 @@
+package com.hazelcast.spi.impl;
+
+import com.hazelcast.nio.Packet;
+import com.hazelcast.spi.Operation;
+
+/**
+ * The OperationScheduler is responsible for scheduling operations to be executed locally.
+ */
+public interface OperationScheduler {
+
+    @Deprecated
```

Comment:
`@Deprecated` seems to be new and `@PrivateAPI` seems to be gone. First change okay, but shouldn't we keep the `@PrivateAPI` in addition?